SuperBuild/External_SimpleITK.cmake

```diff
@@ -79,9 +79,22 @@ ExternalProject_Execute(${proj} \"install\" \"${PYTHON_EXECUTABLE}\" Packaging/s
       -DCMAKE_CXX_FLAGS:STRING=${ep_common_cxx_flags}
       -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}
       -DCMAKE_C_FLAGS:STRING=${ep_common_c_flags}
-      -DCMAKE_CXX_STANDARD:STRING=${CMAKE_CXX_STANDARD}
-      -DCMAKE_CXX_STANDARD_REQUIRED:BOOL=${CMAKE_CXX_STANDARD_REQUIRED}
-      -DCMAKE_CXX_EXTENSIONS:BOOL=${CMAKE_CXX_EXTENSIONS}
+      #
+      # Since SimpleITK always requires C++11, we let its build system
+      # figure out the correct set of flags.
+      #
+      # From Brad:
+      #   "Compiling ITK with C++98 while compiling SimpleITK C++11,
+      #   is not ideal, but all the tests seem to pass for the wrapping.
+      #   Then problem may be if SimpleITK C++ interface was used with
```

Comment:
There may be a problem if the SimpleITK C++ interface is used with this mixing of C++ standards.